src/test/java/com/customer/customerService/CustomerServiceApplicationTests.java

```diff
@@ -1,16 +1,75 @@
 package com.customer.customerService;
 
+import static org.mockito.Mockito.when;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import com.customer.CustomerResponse.CustomerResponse;
+import com.customer.address.Address;
+import com.customer.model.Customer;
+import com.customer.repository.CustomerRepository;
+import com.customer.service.CustomerServiceImpl;
+
+import junit.framework.Assert;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class CustomerServiceApplicationTests {
 
+	@Mock
+	CustomerRepository customerRepository;
+
+	@InjectMocks
+	CustomerServiceImpl customerServiceImpl;
+
+	@Test
+	public void postRequestTest() {
+		// Given
+		Address address = Address.builder().streetName("Abc").state("Haryana").country("France").build();
```

Comment:
can you add couple of more asserts to the unit tests

Reply:
Done , here i am mocking two classes.

Reply:
Done